docs/content/guides/developer/getting-started/sui-install.mdx

```diff
@@ -15,70 +15,102 @@ Sui supports the following operating systems:
-1. Double-click the downloaded file. If the file doesn't automatically expand, manually unzip the file.
-1. Open the expanded folder and double-click the appropriate binary to install, beginning with sui-`<OS>`-`<ARCHITECTURE>`:
+1. Extract all files from the .tgz file into the preferred location on your system. These instructions assume you extract the files into a `sui` folder at the root of your C drive. Replace references to this location in subsequent steps if you choose a different directory.
+
+    :::info 
 
-    - sui-faucet-`<OS>`-`<ARCHITECTURE>`: Local faucet to mint coins on local network.
-    - sui-indexer-`<OS>`-`<ARCHITECTURE>`: An indexer for a local Sui network.
-    - sui-`<OS>`-`<ARCHITECTURE>`: Main Sui binary.
-    - sui-node-`<OS>`-`<ARCHITECTURE>`: Run a local node.
-    - sui-test-validator-`<OS>`-`<ARCHITECTURE>`: Run test validators on a local network for development.
-    - sui-tool-`<OS>`-`<ARCHITECTURE>`: Provides utilities for Sui.
+    Windows does not natively support .tgz files, but you can use a free compressed file app like [7Zip](https://7-zip.org/) to extract.
 
-    The Sui binary takes several minutes to download and install files, so make sure you allocate enough time for it to complete. Actual time to complete depends on your network connection and computer specifications.
+    :::
 
+1. Navigate to the expanded folder. You should have the following extracted files:
+
+    {@include: ../../../snippets/binaries-file-list.mdx}
+
+1. Add the folder containing the extracted files to your `PATH` variable. There are several ways to get to the setting depending on your version of Windows. One way that works on all versions of Windows is to type `sysdm.cpl` in a console to open the System Properties window. Under the **Advanced** tab, click the **Environment Variables...** button.
+1. In the Environment Variables window, select the `Path` variable and click the **Edit...** button. 
+1. In the Edit environment variable window, click **New** and add the path to your expanded folder. Using the example path, this would be `C:\sui`.
+1. Click **OK**.
+
+</TabItem>
+
+</Tabs>
+
+:::info 
+
+Running binaries other than `sui` might require installing prerequisites itemized in the following section. 
+
+:::
 
-### Confirm the installation {#confirm-the-installation}
+#### Confirm the installation {#confirm-the-installation}
 
-To confirm that Sui installed correctly, type `sui` and press Enter. You should see a message about the Sui version installed and help for using Sui commands.
+To confirm that Sui installed correctly, type `sui --version` in your console or terminal and press Enter. The response should provide the Sui version installed. If the console or terminal responds with a command not found error, make sure the full path to your Sui binaries is included in your `PATH` variable.
 
 ## Install from source
```

Comment:
I can't comment on the actual relevant section because it's out of the context of the diff, but the following headings are at the same level as "Install from source" and that makes the section breakdown a little confusing:

- "Install Sui binaries from source"
- "Upgrade Sui binaries"

Can we organise them under "Install from source"?